src/mol-plugin-state/actions/structure.ts

```diff
@@ -121,6 +129,11 @@ const DownloadStructure = StateAction.build({
             asTrajectory = !!src.params.options.asTrajectory;
             format = 'pdb';
             break;
+        case 'alphafolddb':
+            downloadParams = getDownloadParams(src.params.id, id => `https://alphafold.ebi.ac.uk/files/AF-${id.toUpperCase()}-F1-model_v1.cif`, id => `AlphaFold DB: ${id}`, false);
```

Comment:
There are now ``_v2`` files too (and I suppose more will follow). E.g. ``https://alphafold.ebi.ac.uk/entry/Q9UTE2`` => ``https://alphafold.ebi.ac.uk/files/AF-Q9UTE2-F1-model_v2.cif``. 

The URL can be obtained from https://www.alphafold.ebi.ac.uk/api/prediction/Q9UTE2 and they even provide BCIF files!

Reply:
cool, did not knew that api

done, but the bcif files are broken for me, at least molstar throws an error parsing them, so using cif for now

Reply:
Hmm, should probably investigate what's going on there about the bcif files.

Reply:
They will be fixing the BCIF support when people get back from holidays.